Republican 7th ideas change

ComradDoge opened the pull request:
Republics do not deserve absolutism. For the time I thought corruption reduction should suffice but preferably something else should be as a final for republican ideas unless yall think this is fine too.

Also this might be a small change but the change in the reduction of brain cells trying to understand github is much higher

Commit:
Republican 7th ideas change

Republics do not deserve absolutism

stackpoint commented:
I saw your post on discord but forgot to note it down and the nature of the chat format meant it scrolled away.

I agree with the sentiment that it should be replaced but I don't think yearly corruption is a sufficient replacement. Unless republics generate a significant amount of corruption that would require it?

stackpoint reviewed (changes requested):
I saw your post on discord but forgot to note it down and the nature of the chat format meant it scrolled away. It's a lot easier to keep track of issues on GitHub. GitHub can be a bit confusing to get started with but the collaboration element is unrivaled.

I agree with the sentiment that it should be replaced but I don't think yearly corruption is a sufficient replacement. Unless republics generate a significant amount of corruption that would require it?

RayGri commented:
No worries regarding discord if anything forced me to start learning github.

Regarding corruption I will agree that it should be something else but I am not sure what. I was thinking of giving republics a 10% discount of promoting mercantilism but I am not sure how to do that (if that even exists)

stackpoint commented:
Here's a list of modifiers: https://eu4.paradoxwikis.com/Modifier_list

Commit:
Republican Ideas

Moved the rep_tradition to 7th spot and in its place put a diplomat, thematically works for republics and adds up to the 2nd idea of adding relation slots.

Localisation changed in English and German (translate tho,also made it to be in order)

Burgher interaction is also made to accomodate these changes

stackpoint commented on localisation/Idea_Variation_ideas_l_german.yml:
```diff
@@ -143,22 +143,24 @@
- republik1: "Diplomatische Mission"
- republik1_desc: "Wir müssen uns als Republik stärker diplomatisch absichern als manche Monarchien. Denn unsere Art zu Leben stellt Chance und Gefahr zu gleich da."
+ republik5: "Die Macht eines Diplomaten"
+ republik5_desc: "Ein einziger Diplomat am richtigen Ort und zur richtigen Zeit kann den Lauf der Geschichte verändern"
```
Are you German or did you copy/paste this from somewhere else?

stackpoint commented on common/estates/03_burghers.txt:
```diff
@@ -1798,7 +1798,7 @@ estate_burghers =
 		
 		trigger = {
-			has_idea = republik5
+			has_idea = republik7
```
Please be aware that the Jains has this government interaction too.

stackpoint commented:
Adding a diplomat in there seems fine but rearranging the tree has it's own problems as you can see.

Grociu commented on localisation/Idea_Variation_ideas_l_german.yml:
```diff
@@ -143,22 +143,24 @@
- republik7: "Römische Vorväter"
- republik7_desc: "Wir können viel von den Römern lernen, die vor uns kamen. Sie schafften es in Zeiten der Republik ein großes Reich aufzubauen und es relativ zentral zu verwalten."
+ republik1: "Diplomatische Mission"
+ republik1_desc: "Wir müssen uns als Republik stärker diplomatisch absichern als manche Monarchien. Denn unsere Art zu Leben stellt Chance und Gefahr zu gleich da."
```
We are generally no longer updating German localization, only the default English. If this is a lot of extra effort for you, you can safely skip this next time.

Grociu commented on localisation/Idea_Variation_ideas_l_english.yml:
```diff
@@ -145,22 +145,24 @@ l_english:
  republik4_desc: "Although giving a servant a damn good thrashing might be satisfying, it is not the best way to actually get them to work harder. Some natural philosopher has hit on the idea of making profit the prime motivation for people. If we replace monetary rents with payment in kind, our peasants will be incentivized to produce more."
- republik1: "Diplomatic Mission"
- republik1_desc: "We need to play the diplomatic game to ensure our freedom. Because our way of life is both a chance and a threat to our neighbors."
+ republik5: "The Power of a Diplomat"
```
(Suggestion) As a running gag, whenever a function of an idea is changed, I've been using [alliteration](https://en.wikipedia.org/wiki/Alliteration) in individual Idea names when possible. In that vein, here's a list of suggested idea names for this:

- Dynamic Diplomat
- Effective Envoy
- Powerful Plenipotentiary (my current favorite)
- Convincing Conciliator
- Persuasive Peacemaker

... and so on, feel free to come up with one yourself, it's fun.

stackpoint replied:
As long as it's written in English I'm happy.

ComradDoge replied:
Is plenipotentiary even English at this point ? Although I like it for jokes

Grociu replied:
https://www.vocabulary.com/dictionary/plenipotentiary 
Ha, it works well here doesn't it :)

Grociu commented:
Another suggestion for future workflow. I noticed you're pushing from master. Someone taught me when I did my first GitHub collaboration this order for Pull Requests:
- on your local machine copy of the repository, make a new git branch
- commit all changes to that branch locally
- push those changes to your GitHub fork (usually origin)
- submit a Pull Request from your GitHub fork of the repository. 

You'll be merging your own branch into the common master instead of master to master directly. This will assure, that even if the common master branch (the upstream) changes, your branch is truly independent of those changes, and you can still pull the upstream master into your own without trouble. In addition any requested changes after the initial commit you just commit to your branch and they show up in GitHub pull request normally.

I am in no means an expert on the matter and still learning git and GitHub myself, but this is the observed good practice :)

RayGri commented:
Thank you very much. I literally started learning GitHub yesterday so the terms and how everything goes still confuses me but will get used to it with time.

Grociu commented:
Yeah I forget that all these terms are jargon. Here's an article that explains it in plain English, there are probable better resources out there depending on your level of knowledge, but this is an amazing starter:
https://blog.red-badger.com/2016/11/29/gitgithub-in-plain-english
I got to this article by using this course on Odin Project: 
https://www.theodinproject.com/courses/web-development-101/lessons/introduction-to-git

stackpoint commented:
Okay, I will merge this in and modify the Jains
